### PR TITLE
Add automatic generation of mocks.

### PR DESCRIPTION
**Pull request by nikhitashanker77**
-Add automatic generation of mocks.
-Add batch files for running and testing.

### CODE REVIEW

**Comment by nikhitashanker77** [comment]
-Mock files are kept in kubernetes. In this repo I have chosen to not gitignore mock files.
https://github.com/kubernetes/kubernetes/blob/v0.18.0/pkg/kubelet/cadvisor/cadvisor_mock.go 
-In the github workflow I do not regenerate mocks because I think we should be running the tests based on the current state of the files. I think we don't want to generate mock files in the github action because there is an inconsistency between mocks used in the github action and mocks used locally.

File: run.bat [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@echo off
+ 
+:: Usage: Run the BRT service.
+
```
Comment (MichaelBehm) [inline run.bat:4]: add a `go generate` here too

Let's also add a build.bat script

**Review by MichaelBehm — APPROVED** [review]
See my 1 comment

File: run.bat (hunk 1 above)
Comment (nikhitashanker77) [reply run.bat:4]: Added, created build.bat

**Pull request merged**